src/server/src/main/java/io/cassandrareaper/management/http/HttpCassandraManagementProxy.java

```diff
@@ -159,12 +161,27 @@ public String getClusterName() {
 
   @Override
   public List<String> getKeyspaces() {
```

Comment:
Can we please have a unit test for this method? Maybe something that mocks `apiClient.listKeyspaces()` and then just confirms that it was called?

Reply:
@Miles-Garnsey , is this a **todo:** or a **suggestion:** ?

Reply:
Sorry, an issue I think.